_posts/2023-07-20-alert-anomaly-visual.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: "Visualizing alerts and anomalies using OpenSearch Dashboards visualization tools and applications"
+authors: 
+  - jdbright
+  - ashisagr
+  - ohltyler
+  - amgalitz
+  - hnyng
+  - vagimeli
+  - xeniatup
+date: 2023-07-20
+meta_keywords: alerting, anomaly detection, alert, anomaly, observabilty, dashboard, dashboards, visualizations, visualize
+meta_description: Integrating alerts and anomalies visualizations across OpenSearch Dashboards applications.
+twittercard:
+  description: Learn about integrating alerts and anomalies visualizations across OpenSearch Dashboards applications.
+category:
+  - community
+excerpt: 
+---
+
+# Introduction
+
+Prior to OpenSearch 2.9, users maintain state between the data they were viewing in the Discover and Dashboard interfaces and the data needed to create a new anomaly detector or alert monitor. Users provided feedback that creating new detectors or monitors was difficult because of context switching. With OpenSearch 2.9, users who use anomaly detection and alerting can create anomaly detectors and alerting monitors directly from their OpenSearch Dashboards [Vizlib](https://home.vizlib.com/) chart or line visualizations with select aggregations. After creating the new detector or monitor/alert, users can view anomalies or alerts overlaying the configured visualization. For users who have defined detectors or monitors, they can associate their existing detectors or monitors to the visualization, which helps track services that are not specific to the visualization, such as a dependent service.
```

Comment:
Vizlib is incorrect here, maybe you are confusing it with the internal `vislib` library implemented within dashboards. For user's sake, this doesn't need to be explained, and we can keep it simple by just saying "line visualizations".

Reply:
Revised